doc/tutorial/checker.py

```diff
@@ -1,3 +1,10 @@
+from pathlib import Path
+
 def task_checker():
     return {'actions': ["pyflakes sample.py"],
             'file_dep': ["sample.py"]}
+
+def task_checker_pathlib():
```

Comment:
actually I would prefer to have a separate example... to keep examples really simple.
But I merged anyway.